airbyte-webapp/.eslintrc

```diff
@@ -48,6 +48,7 @@
         "ts-expect-error": "allow-with-description"
       }
     ],
+    "@typescript-eslint/consistent-type-definitions": ["error", "interface"],
```

Comment:
:information_source: This enabled the rule and make sure using `type` (when not required) will error out.

Reply:
I understand we autosave but should these minor style rules be tagged as "warn" instead of error?

Reply:
I'd suggest we keep it as "error" if we consider, that we would not want this change to go into the code-base (which formatting missmatches feel like it to me). That said as you mentioned it's having pretty much both the same effect atm, so I don't have extremly strong feelings in either direction.